gradle/libs.versions.toml

```diff
@@ -70,6 +70,8 @@ touchlab-kermit = { module = "co.touchlab:kermit", version = "1.0.0" }
 touchlab-stately = { module = "co.touchlab:stately-common", version = "1.2.1" }
 
 turbine = { module = "app.cash.turbine:turbine", version = "0.7.0" }
+kotlin-test-common = { module = "org.jetbrains.kotlin:kotlin-test-common", version.ref = "kotlin" }
+kotlin-test-annotations-common = { module = "org.jetbrains.kotlin:kotlin-test-annotations-common", version.ref = "kotlin" }
```

Comment:
Doing it this way won't actually import the platform-specific dependency on JVM. Just doing `org.jetbrains.kotlin:kotlin-test` is better because that dependency is multiplatform now.
https://kotlinlang.org/docs/gradle.html#set-dependencies-on-test-libraries

Reply:
Updated to just include kotlin-test. I removed annotations because it appears thats picked up from kotlin-test too, but it's a little hard to tell since everything works either way

Reply:
yeah annotations shouldn't be needed anymore